### ISSUE TITLE
Packaging and distribution

### ISSUE THREAD

**Issue by nocalla** [bank2ynab/bank2ynab#349]
**Is your feature request related to a problem? Please describe.**

#176 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by chamini2** [comment]
Hey, this seems very important to me, making users download a zip file is not ideal IMHO. Would you accept contributions to do this?

**Comment by nocalla** [comment]
Yes please, this is totally beyond my expertise! Even a roadmap of what steps to take would be great.

**Comment by ExalDraen** [comment]
How about using a something like nuitka (https://github.com/Nuitka/Nuitka) to package the whole application into a (OS-specific) executable, and then offer that for download? 

That would be a lot simpler / easier for users, I would think

**Comment by nocalla** [comment]
@ExalDraen that would be good if we can then offer the executable via Github. I haven't packaged before, so no idea on how to go about doing it!

**Comment by limdingwen** [comment]
> Clicking the play button in VS Code no longer works! A sad day, but a small sacrifice.

😆 Well, one could always use `tasks.json`!
